### PR TITLE
docs: ios configuration changes needed for codemagic

### PR DESCRIPTION
**Pull request by GalacticTitan**
## Status

**READY/IN DEVELOPMENT/HOLD**

### CODE REVIEW

**Comment by eseidel** [comment]
Thanks!  @nilsreichardt are you interested/willing to review?  🙃

File: docs/ci/codemagic.mdx [hunk 1]
```diff
@@ -150,6 +174,7 @@ workflows:
           # Add Shorebird to PATH
           echo PATH="$HOME/.shorebird/bin:$PATH" >> $CM_ENV
       - name: 🚀 Shorebird Patch
+        # shorebird patch ios if ios
```
Comment (nilsreichardt) [inline docs/ci/codemagic.mdx:177]:
```suggestion
        # 'shorebird patch ios' if ios
```

File: docs/ci/codemagic.mdx [hunk 2]
```diff
@@ -107,7 +111,19 @@ workflows:
...
-2. Add a script to your workflow to set up Shorebird. This script will install
+2. If your target is `ios`, you need to specify the code signing mechanism. First you need to setup provisional profile and certificate for that. You may find the setup [here](https://docs.codemagic.io/yaml-code-signing/signing-ios/).
+
+```yaml
+workflows:
+  example:
+    name: Example
+    environment:
+      ios_signing:
+        distribution_type: ad_hoc #app_store # or | development | enterprise
```
Comment (nilsreichardt) [inline docs/ci/codemagic.mdx:122]:
```suggestion
        distribution_type: ad_hoc # or app_store | development | enterprise
```

File: docs/ci/codemagic.mdx [hunk 3]
```diff
@@ -138,8 +154,16 @@ workflows:
   example:
     name: Example
     environment:
+      # if target is ios use below signing configuration
+      ios_signing:
+       distribution_type: ad_hoc #app_store # or | development | enterprise
```
Comment (nilsreichardt) [inline docs/ci/codemagic.mdx:159]:
```suggestion
       distribution_type: ad_hoc # or app_store | development | enterprise
```

File: docs/ci/codemagic.mdx [hunk 4]
```diff
@@ -138,8 +154,16 @@ workflows:
...
+       bundle_identifier: your package name # for wildcard provisional profiles
       groups:
         - shorebird
+      # if your target is ios add signing configuration
+      ios_signing:
+        distribution_type: ad_hoc #app_store # or: ad_hoc | development | enterprise
```
Comment (nilsreichardt) [inline docs/ci/codemagic.mdx:165]:
```suggestion
        distribution_type: ad_hoc # or app_store | development | enterprise
```

**Comment by GalacticTitan** [comment]
@nilsreichardt Thanks for reviewing. I have made suggested changes.

**Review by nilsreichardt — APPROVED** [review]
Thanks! LGTM

**Comment by GalacticTitan** [comment]
@eseidel sorry, forgot check with prettier. I am new to this.

**Review by felangel — APPROVED** [review]

**Pull request merged**